evm/src/cpu/kernel/interpreter.rs

```diff
@@ -1542,20 +1226,51 @@ mod tests {
         //         PUSH1 0x42
         //         PUSH1 0x27
         //         MSTORE8
-        let code = vec![
+        let code = [
```

Comment:
Might as well keep this as a `Vec` yeah?

Reply:
Clippy was complaining there, which is why I removed it.